packages/cli/lib/s3.ts

```diff
@@ -29,3 +30,33 @@ export async function openFile (this: any, csvw: Csvw.Mapping, s3Endpoint: strin
 
   throw new Error(`Could not read file "${csvw.url}" from S3. It was neither Buffer or Readable`)
 }
+
+export function uploadFile (this: any, fileName: string, s3Endpoint: string, s3Bucket: string) {
+  const quadStream = new PassThrough()
+
+  const s3 = new aws.S3({
+    endpoint: s3Endpoint,
+  })
+
+  s3.upload({
+    Bucket: s3Bucket,
+    Key: fileName,
+    Body: quadStream,
+  }).on('httpUploadProgress', progress => {
+    this.log.info(`Uploaded ${progress.loaded / 1024} of ${progress.total / 1024 || '?'} kB`)
```

Comment:
@bergos do you know why the `log` call in the upload callbacks are not being printed to the output? as if they were being called too late and they are lost before the process ends?